types/web-bluetooth/index.d.ts

```diff
@@ -98,11 +101,9 @@ interface BluetoothDevice extends EventTarget, BluetoothDeviceEventHandlers, Cha
 	readonly name?: string;
 	readonly gatt?: BluetoothRemoteGATTServer;
 	readonly uuids?: string[];
-
```

Comment:
any special reason for removing the blank lines?

Reply:
Not other than to be consistent with the other part of the files. Happy to put them back in :)